spec/collectionspace/converter/core/organization_spec.rb

```diff
@@ -5,65 +5,86 @@
-    "/document/*/orgTermGroupList/orgTermGroup/termStatus",
-    { xpath: "/document/*/orgTermGroupList/orgTermGroup/termFlag", transform: ->(text) { CSURN.parse(text)[:label].downcase } },
-    { xpath: "/document/*/dissolutionDateGroup/dateDisplayDate", transform: ->(text) { text.split('-')[0] } },
-    { xpath: "/document/*/foundingDateGroup/dateDisplayDate", transform: ->(text) { text.split('-')[0] } },
-    "/document/*/foundingPlace",
-    { xpath: "/document/*/contactGroupList/contactGroup/contactEndDateGroup/dateDisplayDate", transform: ->(text) { text.split('-')[0] } },
-    { xpath: "/document/*/contactGroupList/contactGroup/contactDateGroup/dateDisplayDate", transform: ->(text) { text.split('-')[0] } },
-    { xpath: "/document/*/contactGroupList/contactGroup/contactStatus", transform: ->(text) { CSURN.parse(text)[:label].downcase } },
-    { xpath: "/document/*/contactGroupList/contactGroup/contactName", transform: ->(text) { CSURN.parse(text)[:label] } },
-    { xpath: "/document/*/contactGroupList/contactGroup/contactRole", transform: ->(text) { CSURN.parse(text)[:label] } },
-    { xpath: "/document/*/organizationRecordTypes/organizationRecordType", transform: ->(text) { CSURN.parse(text)[:label].downcase } },
-    "/document/*/groups/group",
-    "/document/*/functions/function",
-    "/document/*/historyNotes/historyNote",
-    "/document/*/emailGroupList/emailGroup/email",
-    "/document/*/emailGroupList/emailGroup/emailType",
-    "/document/*/telephoneNumberGroupList/telephoneNumberGroup/telephoneNumber",
-    "/document/*/telephoneNumberGroupList/telephoneNumberGroup/telephoneNumberType",
-    "/document/*/faxNumberGroupList/faxNumberGroup/faxNumber",
-    "/document/*/faxNumberGroupList/faxNumberGroup/faxNumberType",
-    "/document/*/webAddressGroupList/webAddressGroup/webAddress",
-    "/document/*/webAddressGroupList/webAddressGroup/webAddressType",
-    "/document/*/addressGroupList/addressGroup/addressType",
-    "/document/*/addressGroupList/addressGroup/addressPlace1",
-    "/document/*/addressGroupList/addressGroup/addressPlace2",
-    "/document/*/addressGroupList/addressGroup/addressMunicipality",
-    "/document/*/addressGroupList/addressGroup/addressStateOrProvince",
-    "/document/*/addressGroupList/addressGroup/addressPostCode",
-    "/document/*/addressGroupList/addressGroup/addressCountry",
-  ]}
 
-  context 'For maximally populuated record' do
-    it "Maps attributes correctly" do
-      test_converter(doc, record, xpaths)
-    end
-  end
-  context 'For minimally populated record' do
-    let(:attributes) { get_attributes_by_row('core', 'authorg_core_all.csv', 15) }
-    let(:doc) { get_doc(coreorganization) }
-    let(:record) { get_fixture('core_organization_row15.xml') }
-    let(:xpath_required) {[
-      "/document/*/orgTermGroupList/orgTermGroup/termDisplayName"
-    ]}
+  describe 'map_common' do
+    p = '/document/organizations_common'
+    context 'For maximally populuated record' do
```

Comment:
@Paulmulonzia - In the spec files here, I've started doing a new pattern for tests. I like these better because A) if multiple fields fail, I get an accurate number of failures and a failure report on each, rather than only one failure if a whole test on a bunch of xpaths fails; B) for the tests of xpaths with URNs, B1) I don't have to mess with manually specifying out each possible value with the xpath-transform lambda in my xpaths array; B2) This tests that the value and vocabulary type or authority type+subtype is correct with one test; B3) Failure is more informative and actually shows you what the problem was.

@mark-cooper - I could not figure out how to helper-ize these test patterns. I had to have the `it ... do` in each spec file's code in order to use the variables and methods used up in the `let` section. Any ideas?